android/res/color/preference_summary_text.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
At some point we started adding Apache licenses, I can't remember if we were continuing to change to Apache.

Files we have written from scratch should have no issue becoming Apache, however, files from other projects we can't change the header.

As an author, I'd leave it up to you to decide.

I think Apache suits best for things like utility classes, libraries (like jlibtorrent, but we license that one under MIT), but things that are very specific to FrostWire that some other competing client could copy we should probably leave as GNU to keep the code free.

I'm also confused to be honest, @aldenml knows better what was that decision.